layouts/shortcodes/release-data.html

```diff
@@ -1,6 +1,6 @@
 {{ range $data := .Site.Data.releases.schedule.schedules }}
-{{- $dataVersion := printf "%.2f" $data.release  -}}
-{{- $dataIdVersion := replace ( printf "%.2f" $data.release ) "." "-" | anchorize -}}
+{{- $dataVersion := $data.release -}}
+{{- $dataIdVersion := replace ( $data.release ) "." "-" | anchorize -}}
```

Comment:
Ah, with the format change here: please leave the `printf` in place; it's validating that we have something that looks like a float (or, in other words, that it's a major.minor and not a patch version).

Reply:
Yep that's now a string, keeping the `printf`.